libs/remix-ui/workspace/src/lib/actions/workspace.ts

```diff
@@ -41,17 +40,17 @@ export const addInputField = async (type: 'file' | 'folder', path: string, cb?:
   return promise
 }
 
-export const createWorkspace = async (workspaceName: string, isEmpty = false, cb?: (err: Error, result?: string | number | boolean | Record<string, any>) => void) => {
+export const createWorkspace = async (workspaceName: string, workspaceTemplateName, isEmpty = false, cb?: (err: Error, result?: string | number | boolean | Record<string, any>) => void) => {
```

Comment:
`workspaceTemplateName` is a string ? can you put the type?

Reply:
fixed